### PR TITLE
[TIG-770] (Rework) Styling and layout for signup container

### PR DESCRIPTION
**Pull request by stephenbelyea**
## Ticket(s)

* [TIG-770](https://app.clickup.com/t/9009201449/TIG-770) - Original task (mostly complete)
  * [TIG-792](https://app.clickup.com/t/9009201449/TIG-792) - Focused sub-task 
* [TIG-759](https://app.clickup.com/t/9009201449/TIG-759) - Secondary task (partially complete?)

## Description

This PR extracts changes from @klancashire's [original branch](https://github.com/TakingITGlobal/commit2act/pull/116) which ran into some merge conflict troubles. Changes have mostly been ported over to this one, with the exception of the Next/Back buttons - as there was considerable Link/Button rendering work done upstream which did not survive the merge!

Please refer to the original PR for further context - minus the Next/Back buttons.

## Testing

1. Pull down and install the branch (`npm install`)
2. Run tests locally (`npm run test:basic`)
3. Run Storybook locally (`npm run storybook`)
4. View Components > LinearProgress and confirm it renders matching designs
5. View Components > Paper and confirm the gradient matches designs
6. View Views > SignupContainer and confirm the gradient background is applied

### CODE REVIEW

File: src/components/Navbar.js
Comment (klancashire) [inline src/components/Navbar.js:1]: This is just a random clean up thing that made it into this PR somehow, me with my paws in everything lol

File: src/styles/page-container.js
Comment (klancashire) [inline src/styles/page-container.js:1]: A note for @FayeMao or @kirstenjvd when you work on the typography for this one, that I have started some signup container specifics in the page container file. Ideally they will not be here forever but this was the best place I had to put them that would require regularly restyling on components and hopefully this file will die in a future rework.

**Review by klancashire — APPROVED** [review]
A much nicer clean up on my file. A little low key bummed to lose all the work integrating the components into the signup container because I struggled but alas, wasn't in the ticket and was riddled with merge issues so this is for the best. Hopefully integrating them again will go smoother from here. :)

**Review by FayeMao — APPROVED** [review]
Passes all the tests + linear progress, paper and gradient background is applied. Looks good!

**Review by AveryDodd — APPROVED** [review]

**Pull request merged**